assets/templates/datasetLandingPage/static.tmpl

```diff
@@ -76,43 +76,43 @@
 
 <div class="meta-wrap">
     <div class="wrapper">
-        <dl class="col-wrap">
+        <div class="col-wrap">
```

Comment:
This change here seems to make the screenreader read everything within the div container when you first land on it from the breadcrumbs above. When you move to the next link (the contact), and then move back to the ONS statistics logo, it then only reads the image element info. Is this the intended behaviour?